src/ngx_http_lua_util.h

```diff
@@ -69,6 +69,10 @@ extern char ngx_http_lua_headers_metatable_key;
 #define NGX_HTTP_SWITCHING_PROTOCOLS 101
 #endif
 
+#ifndef NGX_HTTP_NOT_MODIFIED
```

Comment:
We don't need this because the macro is always there (as far as the minimum nginx core version that ngx_lua supports, 0.8.54).